templates/example.html

```diff
@@ -75,5 +75,12 @@
         {{/shapes}}
       </ul>
     </section>
+    <section>
+      <a href="https://vercel.com/?utm_source=onfido-oss&utm_campaign=oss"
+        ><img
```

Comment:
I believe it did auto-format here, however this should also be fine for Prettier, but feels a bit better:

```suggestion
      <a href="https://vercel.com/?utm_source=onfido-oss&utm_campaign=oss">
        <img
```

Reply:
It did, yeah. Good suggestion :)